spec/controllers/openid_connect/authorization_controller_spec.rb

```diff
@@ -99,16 +99,21 @@
 
     context 'user is not signed in' do
       it 'redirects to login' do
```

Comment:
update the comment?

`it 'redirects to login with the request_id'` or something?